db/migrate/20250218191831_create_meal_restrictions.rb

```diff
@@ -0,0 +1,13 @@
+class CreateMealRestrictions < ActiveRecord::Migration[7.0]
+  def change
+    create_table :meal_restrictions do |t|
+      t.string :contains, null: false
+      t.string :absence, null: false
+      t.boolean :enabled, default: true, null: false
```

Comment:
There is a pattern in the app of having a deactivated_at column instead of this. See db/migrate/20191225183248_add_deactivated_at_to_groups.rb for instance. If you do it this way you'll get some stuff for free.